conf/application.conf

```diff
@@ -165,7 +165,7 @@ contact-frontend {
 
 mongodb {
   uri = "mongodb://localhost:27017/voa-bar-xml-frontend?rm.tcpNoDelay=true&rm.nbChannelsPerNode=3"
-  timeToLiveInSeconds = 3600
+  timeToLiveInSeconds = 72000 // 20 hours in seconds same as sign out timeout
```

Comment:
Thats perfect, but I'm little bit worried that index won't be updated because it already exist.
`reactivemongo.api.indexes.DefaultCollectionIndexesManager#ensure` Compare only index name. Maybe we need to delete odl one and create new one.

Reply:
Done in https://github.com/hmrc/voa-bar-xml-frontend/pull/182/commits/2a1e528a871c6d0270c70d366b0a14edf89f91b2